src/bulk/common.ts

```diff
@@ -161,6 +161,8 @@ export class Batch {
 export class BulkWriteResult {
   result: BulkResult;
```

Comment:
This still has the undefined ids issue, I think they're not being added in correctly?
I reran the snippet here:
https://jira.mongodb.org/browse/NODE-2923

Reply:
That was fixed in this PR: https://github.com/mongodb/node-mongodb-native/pull/2653